docker-compose.yml

```diff
@@ -14,6 +14,7 @@ services:
     build: ./active-sessions 
     ports: 
       - "8001:8001"
+      - "50052:50052"
```

Comment:
just out of curiosity, what is the other exposed port used for?

Reply:
Dont know but the one i add is to communicate with other services